### DISCUSSION TITLE
Should wlsunset be running after start?

### DISCUSSION THREAD

**Discussion by heapifyman** [Manjaro-Sway/manjaro-sway#433] [Q&A] [answered]
[99-autostart-applications.conf](https://github.com/manjaro-sway/desktop-settings/blob/sway/community/sway/etc/sway/config.d/99-autostart-applications.conf) calls `$wlsunset` on login.

[https://github.com/manjaro-sway/desktop-settings/blob/sway/community/sway/etc/sway/definitions](https://github.com/manjaro-sway/desktop-settings/blob/sway/community/sway/etc/sway/c) defines `$wlsunset` as `set $wlsunset '[ -x "$(command -v wlsunset)" ] && /usr/share/sway/scripts/sunset.sh "on"'`

But if I understand correctly, [sunset.sh](https://github.com/manjaro-sway/desktop-settings/blob/sway/community/sway/usr/share/sway/scripts/sunset.sh) does accept any parameter `on`.

And if I call `[ -x "$(command -v wlsunset)" ] && /usr/share/sway/scripts/sunset.sh "on"` manually it returns `{"alt":"off","tooltip":"no gamma correction"}`

If I remember correctly, wlsunset used to be enabled automatically after login.

Am I missing something?

**Comment by boredland** [comment, marked as answer]
Yes, I removed that behavior by accident. Will patch in a sec.
